cmd/cri-containerd/cri_containerd.go

```diff
@@ -43,9 +43,9 @@ func main() {
 		glog.Exitf("Failed to connect containerd socket %q: %v", o.ContainerdSocketPath, err)
 	}
 
-	glog.V(2).Infof("Run cri-containerd grpc server on socket %q", o.CRIContainerdSocketPath)
-	service := server.NewCRIContainerdService(conn)
-	s := server.NewCRIContainerdServer(o.CRIContainerdSocketPath, service, service)
+	glog.V(2).Infof("Run cri-containerd grpc server on socket %q", o.SocketPath)
```

Comment:
think I liked CRIContainerdSocketPath better.. when you see socket path makes me wonder socket path for..?